packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/listFeatures.ts

```diff
@@ -199,6 +225,16 @@ function getListChains(editor: IEditor) {
     return VListChain.createListChains(editor.getSelectedRegions());
 }
 
+function getCacheNextSibiling(event: PluginKeyboardEvent, editor: IEditor) {
+    const element = cacheGetEventData(event, 'nextSibiling', () => {
+        const range = editor.getSelectionRange();
+        const pos = Position.getEnd(range).normalize();
+        const traverser = editor.getBodyTraverser(pos.node);
```

Comment:
Better do null check for traverser and traverser.getNextBlockElement(). In same cases they can return null.